### PR TITLE
Fix outstanding tagger issues

### LINKED ISSUE #903: [Bug Report] Match fingerprints causes UI crash when >100 scenes shown
**Describe the bug**
UI displays the error message: `Unhandled Rejection (Error): response error: Too many fingerprints.`

**To Reproduce**
Open the scene tagger view, setting page size to 120. Click match fingerprints button.

**Expected behavior**
Any errors from the match fingerprint operation should be handled cleanly.

Ideally where there are more than 100 scenes displayed, the tagger should be performing a series of queries, rather than everything at once.

### PR DESCRIPTION
**Pull request by InfiniteTF**
* Fixes issue where users could sometimes not save due to a state race-condition.
* Fixes duration sorting so closest match is sorted first.
* Adds is_missing stash_id filter for scene/performer/studio.
* Paginates fingerprint lookups so users can search for arbitrary amount of scenes.
* Disable `Match Fingerprint` button after search is performed, and show number of matches.

Resolves #903, #893, #915

### CODE REVIEW

**Review by WithoutPants — APPROVED** [review]

**Comment by WithoutPants** [comment]
Just discovered: UI is still not handling error conditions when searching scenes by fingerprint or search. 

To reproduce, show the tagger view for some scenes. Kill the stash process, then click Search or Match fingerprints. Causes UI stack trace to show.

**Comment by InfiniteTF** [comment]
> Just discovered: UI is still not handling error conditions when searching scenes by fingerprint or search.

Thanks for the heads up, I forgot to handle network errors. Should be resolved now.

**Comment by WithoutPants** [comment]
I've just done some minor changes to show the network error under the specific scene being searched, rather than at the top.

**Comment by InfiniteTF** [comment]
> I've just done some minor changes to show the network error under the specific scene being searched, rather than at the top.

Looks good. I just tweaked it a bit so scene network errors aren't cleared on fingerprint search, and vice versa.

File: ui/v2.5/src/components/Tagger/Tagger.tsx [hunk 1]
```diff
@@ -168,9 +167,6 @@ const TaggerList: React.FC<ITaggerListProps> = ({
       .filter((s) => s.stash_ids.length === 0)
       .map((s) => s.id);
 
-    // clear search errors
-    setSearchErrors({});
```
Comment (WithoutPants) [inline ui/v2.5/src/components/Tagger/Tagger.tsx]: I'm not sure why this was done. With this change, if the text search fails but the fingerprint search does not, then it still shows the error message and does not show the fingerprint matched scene.

Comment (InfiniteTF) [reply ui/v2.5/src/components/Tagger/Tagger.tsx]: The issue was that if a fingerprint wasn't found the scene state would flip from `network error` to `no results found` despite no search having been performed. I hadn't considered your scenario though. I've corrected this now so the scene results are wiped if the query fails, and reinstated the search error clearing.

**Comment by InfiniteTF** [comment]
Added a small fix to not abort if the user tries to submit fingerprints for a scene that for some reason doesn't exist. This could easily happen for a number of different reasons, and would basically force the user to wipe the fingerprint cache. Long term the submission state should be moved to the db so it can be properly cleaned up, and persist across browsers.

**Review by WithoutPants — APPROVED** [review]

**Pull request merged**